mrt/meetings/participant.py

```diff
@@ -60,8 +60,10 @@ class Participants(PermissionRequiredMixin, MethodView):
     def get(self):
         Form = custom_form_factory(self.form_class)
         form = Form()
+        participants = Participant.query.current_meeting().participants()
         return render_template('meetings/participant/participant/list.html',
-                               form=form)
+                               form=form,
+                               participants=participants)
```

Comment:
Why are you including this variable in the context? It's not used in the template.